VirtualMachineSetup.md

```diff
@@ -0,0 +1,152 @@
+
+2. Download [Debian 11.2](https://www.debian.org/releases/bullseye/debian-installer/), bullseye. Download the `netinst CD image` appropriate for your processor. If your computer is has an ARM based processor, such as the Macbooks with M1 chips you **must** download the `arm64` image, otherwise you probably want the `amd64` image.
+
+3. Create a new virtual machine in Parallels (File -> New). Chose to install an OS from a image file and navigate to the `.iso` image you downloaded in step 2. 
+
+4. Name your VM whatever you would like, and check the box to *Customize settings before installation*. Then click *Create*.
+
+![Install Page - Name & Location](setup_screenshots/name_and_location.png?raw=true)
+
+5. Customize the settings to your liking. Some suggestions:
+
+ - **Options/Sharing:** You can keep a shared folder between your VM and your home computer. This can be useful to transfer documents back and forth, etc. You can configure this how you wish, I recommend clicking Custom Folders... and creating a single shared folder.
+
+- **Hardware/CPU & Memory:** Change Memory to at least 8GB. 16GB or more is preferred if your computer has enough RAM. 2 processors should be fine, but if you feel the need you can increase that as well.
+ 
+ - **Hardware/Shared Printers:** Uncheck *Share Mac printers with Debian GNU/Linux*.
+
+ - **Hardware/Sound & Camera:** Uncheck *Share Mac camera with Linux*
+
+ - **Backup** Check the box for *SmartGuard* in order to enable automatic snapshots of your VM. This is highly recommended as long as you have the storage space. Note that you can also manually take snapshots at any time. Click *Details...* to customize how often snapshots are taken and how many are kept on your machine. If you aren't sure just keep the default settings.
+
+
+Other settings that may not be present in newer versions of Parallels (in which case you can safely ignore)
+
+- **Options/Applications:**  un-check Share Linux applications with Mac unless you enjoy clutter or have a specific need.
+- **Options/More Options:** check Authenticate with macOS SSH public key.
+
+When you are done, close the Configuration modal, and click *Continue*
+
+## Installing Debian 
+
+Once you boot your new virtual machine in Debian you will be prompted with the install guide. 
+
+1. Chose either *Install* or *Graphical Install*, the latter will be used in this guide but it is just a slightly nicer UI, all of the steps and options are the same. 
+
+![Debian Install Screen](setup_screenshots/install_screen.png?raw=true)
+
+2. Select your language, location, and keyboard layout on the next few screens. Debian will then begin installing. 
+
+3. *Configure the network* You will be asked for a Hostname. Since we aren't running a network this doesn't really matter, you can leave the default `debian`.
```

Comment:
This is another difference between prod and dev: on prod we need to think about what this name is, since LVM uses it for naming logical volumes. I've been using `VxMark`, tho I'm gonna switch it to `Vx` for use on all prod machines. Again, I'm interested to hear with @benadida and @eventualbuddha think about that. We could just use the default `debian` on prod, too, it's not ultimately _that_ important...